### DISCUSSION TITLE
[Bug]: Critical fault #11

### DISCUSSION THREAD

**Discussion by uguraltinsoy** [meshtastic/firmware#4532] [Troubleshooting]
### Category

Other

### Hardware

DIY

### Firmware Version

2.4.2.5b45303

### Description

I made the connections as shown below and completed the setup, but I am encountering an error during installation. Despite the error, the device powers on, but I get a "Critical fault #11" error, and I cannot complete the setup. I tried two different ESP32s and two different Ra-02 modules, so it's unlikely to be a hardware-related issue.

https://adrelien.com/blog/meshtastic-diy-how-to-build-your-own-meshtatic-node-esp32-lora-radio/amp/
![chrome_qdfiUw7vtH](https://github.com/user-attachments/assets/f4a2a2c1-c687-4894-a0ab-23384fb3c88b)

![opera_IUx4HWyqHs](https://github.com/user-attachments/assets/9d2ec581-a949-4e56-ad26-620ea06c7e78)
![WhatsApp Image 2024-08-21 at 14 06 03](https://github.com/user-attachments/assets/b153fc88-1bcc-414d-a28f-a47578980afe)
![WhatsApp Image 2024-08-21 at 14 06 01](https://github.com/user-attachments/assets/84bfb5a1-8807-46a6-9b08-3d483b7a38ea)

**Comment by Lolagatorade** [comment]
Had the error too but my device was functional after a reboot and it was only a one time error. 

The error came up randomly during operation roughly 17-20hr after the update

**Comment by Lolagatorade** [comment]
![image](https://github.com/user-attachments/assets/4e2058b1-836b-4686-bfb8-18b93ecdeeb6)

**Comment by uguraltinsoy** [comment]
I made the connection according to the table here and the problem disappeared

MISO -> IO19
MOSI -> IO27
SCK -> IO5
NSS -> IO18
RST -> IO23
DIO0 -> IO26
DIO1 -> IO33
DIO2 -> IO32
